src/components/Details/index.js

```diff
@@ -17,7 +18,10 @@ export default function Details() {
 		<StyledDetails>
 			<h1>{meal.title}</h1>
 			<BackButton />
-			<EditButton />
+			<div>
+				<EditButton />
+				<DeleteButton id={meal.id} />
```

Comment:
It could be better to use an onclick event at this point instead to lift the id down and use the onclick inside the button component. The background is to keep components as simple as possible.

Reply:
Thanks, Stefan! 
I was still unsure where the onClick functions are supposed to be placed with components. Now it is clearer.